packages/sources/gemini/test/unit/adapter.test.ts

```diff
@@ -9,12 +9,16 @@ describe('execute', () => {
   const execute = makeExecute()
 
   describe('validation error', () => {
-    const requests = [{ name: 'empty body', testData: {} }]
```

Comment:
According to the README, this EA should still be able to have a unit test on empty body, as it requires `token` to be set.

Reply:
`empty body` check will make a DP call because it has default value, it will use it.

Reply:
I see. Could you update the schema to make it not-required since it has a default?

Reply:
I don't really understand why it has both `required` and `default` specified, doesn't make sense to me